### PR TITLE
By inserted at

### PR DESCRIPTION
**Pull request by marinakr**

### CODE REVIEW

File: apps/deduplication/lib/deduplication/v2/match.ex [hunk 1]
```diff
@@ -112,7 +90,11 @@ defmodule Deduplication.V2.Match do
     log_insert(merge_candidates, system_user_id)
   end
 
-  defp log_insert(merge_candidates, system_user_id) do
+  def score(score) do
+    if is_integer(score), do: score / 1, else: score
```
[the remark this answers is not in the document]
Comment (marinakr) [reply apps/deduplication/lib/deduplication/v2/match.ex]: match 0 and 1?

File: apps/deduplication/lib/application.ex [hunk 2]
```diff
@@ -1,37 +1,50 @@
...
+  alias Deduplication.Consumer
+  alias Deduplication.Producer
   alias Deduplication.V2.PythonWorker
-  alias Deduplication.Worker
 
   def start(_type, _args) do
-    gen_consumer_impl = Deduplication.Kafka.GenConsumer
-    consumer_group_name = Application.fetch_env!(:kafka_ex, :consumer_group)
-    topic_names = ["deactivate_person_events"]
+    pollboy_supervisor =
```
[the remark this answers is not in the document]
Comment (marinakr) [reply apps/deduplication/lib/application.ex]: its for python

File: apps/core/priv/repo/migrations/20181211132525_addresses_text.exs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      modify(:apartment, :text)
+      modify(:zip, :text)
+      add(:person_first_name, :text)
+      add(:person_last_name, :text)
+    end
+
+    create(index(:person_addresses, [:updated_at]))
+    create(index(:person_addresses, [:settlement_id, :person_first_name, :person_id]))
+    create(index(:person_addresses, [:settlement_id, :person_last_name, :person_id]))
+    drop(index(:person_addresses, [:settlement_id, :person_id]))
```
[the remark this answers is not in the document]
Comment (marinakr) [reply apps/core/priv/repo/migrations/20181211132525_addresses_text.exs]: table is empty on this moment

File: apps/deduplication/lib/deduplication/v2/match.ex (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlexKovalevych) [reply apps/deduplication/lib/deduplication/v2/match.ex]:
```
def score (score) when is_integer(score), do: score/1
def score(score), do: score
```

File: apps/deduplication/lib/application.ex (hunk 2 above)
[the remark this answers is not in the document]
Comment (AlexKovalevych) [reply apps/deduplication/lib/application.ex]: poolboy vs pollboy

**Pull request merged**